api/incidents/incidentsRouter.js

```diff
@@ -365,60 +95,6 @@ router.get('/incident/:id', async (req, res) => {
   }
 });
 
-// ### POST /createincidents ###
-// - returns success / error response message from BE
-// ⬇️ swagger docs code generation ⬇️
-// NOT COMPLETE <--> TODO: @NIC
-/**
- * @swagger
- * /incidents/createincidents:
- *  post:
- *    summary: SHOULDN"T BE USED (NIC) ... path to add an incident to the the database
- *    tags:
- *      - incidents
- *    produces:
- *      - application/json
- *    responses:
- *      201:
- *        description: Server response success
- *        content:
- *          application/json:
- *            schema:
- *              type: object
- *              required:
- *                - api
- *              properties:
- *                 message:
- *                    type: string
- *                    example: "Success"
- *      500:
- *        description: Server response error
- *        content:
- *          application/json:
- *            schema:
- *              type: object
- *              required:
- *                -api
- *              properties:
- *                message:
- *                  type: string
- *                  example: "Request Error"
- */
-
-/**
- * Returns a CSV of incidents
- * By default returns all incidents
- * Optional query string can limit responses by date range or state
- *
- * Possible Query Strings:
- * /download?state=*StateName Here*
- * /download?startDate=*StartDate Here*&endDate=*End Date Here*
- * /download?state=*StateName Here*&start=*StartDate Here*&end=*End Date Here*
- * State name ex: "New York" default: null
- * Start Date ex: "05-13-2020" default: One year ago from Today
- * End Date ex: "12-04-2020" default: Today
- */
-
 router.get('/download', async (req, res) => {
```

Comment:
Looks like we'll have to get a download route in our new route.

Reply:
I was under the impression that the download function in the frontend simply convert the array of objects stored in local memory into a csv file?

Reply:
I'll  have to look into this as well